### PR TITLE
update style.css

### PR DESCRIPTION
**Pull request by mileamadalin**
@rodica-andronache @ineagu 
I made two small changes for a better fit on the responsive part. So I changed the width at 320px from 400px on line 1191 and I comment the line 125 for word-spacing 10px (if you want we can delete this line for mobile version).

### CODE REVIEW

**Comment by ineagu** [comment]
looks better, the only issue now is the message : update style.css , it should say what exactly you did as short as possible like : Fixed about section responsive issues.

Much better will be to first open an issue with the problem, then sending the pullrequest fixing the issue, so the message will be : Fixed about section responsive issues #xx

File: style.css [hunk 1]
```diff
@@ -1188,7 +1188,7 @@ figure.effect-portfolio:hover img {
 #about h2 {
     font-family: Futura, "Trebuchet MS", Arial, sans-serif;
     font-size: 44px;
-    width: 400px;
+    width: 320px;
```
Comment (mileamadalin) [inline style.css]: Am reverificat si aici cel mai bine ar fi sa facem 277px. La 320 este totusi cam la limita, deci in loc de 320px trebuie redus la 277px.

**Comment by mileamadalin** [comment]
Ok, I get it :D

**Comment by ineagu** [comment]
ok, atunci ar trebui trimis alt pull request.

**Pull request closed without merging**